pyodide_build/cli/skeleton.py

```diff
@@ -19,6 +19,70 @@ def callback() -> None:
     return
 
 
+def get_recipe_dir(recipe_dir: str | None) -> Path:
+    if recipe_dir:
+        return Path(recipe_dir)
+    cwd = Path.cwd()
+    root = build_env.search_pyodide_root(curdir=cwd)
+
+    if not root:
+        root = cwd
+
+    return root / "packages"
+
+
+@app.command("enable")
```

Comment:
As `_disabled` flag is private, how about hiding the command from the users?

```suggestion
@app.command("enable", hidden=True)
```

Reply:
How about making the `_disabled` flag not private?

Reply:
Sounds reasonable to me. But lets handle it in a separate PR